community.json

```diff
@@ -124,6 +124,12 @@
         "state": "working",
         "url": "https://github.com/YunoHost-Apps/couchpotato_ynh"
     },
+    "Cryptpad": {
```

Comment:
The name doesn't respect [YEP 1.1](https://github.com/YunoHost/doc/blob/master/packaging_apps_guidelines_fr.md#yep-11---nommer-son-app-et-son-d%C3%A9p%C3%B4t---valid%C3%A9--manuel--notworking-) on package naming.